global-test.openshift.properties

```diff
@@ -7,8 +7,8 @@
 ###############################################
 
 # Databases
-intersmash.mysql.image=quay.io/centos7/mysql-80-centos7:centos7
-intersmash.postgresql.image=quay.io/centos7/postgresql-13-centos7:centos7
+intersmash.mysql.image=registry.redhat.io/rhel10/mysql-84:latest
```

Comment:
@fabiobrz same as above: is that intended? I mean switching from quay.io to registry.redhat.io ...